Source/Resources/Content/MSGraph/Groups/BUILTIN - Microsoft 365 Users - Preview.json

```diff
@@ -2,7 +2,7 @@
   "assignedLicenses": {
     "$ref": "ResourceContext:M365Licenses"
   },
-  "description": "This manual group is used to assign Microsoft 365 licenses to preview only users for EMS and O365 functionality.",
+  "description": "What is it: A manual group of users that are in the preview ring. \n Simeon Notes: This group can be used to apply a user policy to a subset of users before deploying to all users.",
```

Comment:
**What does this do?**
Creates a manual group used to assign Microsoft 365 licenses. This group can be used to release changes to a subset of users before deploying to all users.

**Why should you use this?**
If you want to test policies in release rings to a subset of users before deploying to all users. 

**What is the end user impact?**
Users of this group can receive configuration changes before others users. 

**Learn more**
NA